jbang-catalog.json

```diff
@@ -0,0 +1,12 @@
+{
+  "catalogs": {},
+  "aliases": {
+    "wanaku": {
+      "script-ref": "jbang/src/main/java/ai/wanaku/jbang/WanakuJBang.java",
+      "dependencies": [
+        "ai.wanaku:cli:${wanaku.version:0.0.3}"
```

Comment:
@orpiske This would be good to be updated with each release, right ? For that, i need to understand more, how is the release done now. But i guess we can live with manual bumping for some early time.

Reply:
We can use a resource filter like this:

1. Adjust the pom: https://github.com/wanaku-ai/wanaku/blob/main/core/core-util/pom.xml#L50-L58
2. Replace the fixed version with `${project.version}`, like here: https://github.com/wanaku-ai/wanaku/blob/main/core/core-util/src/main/resources/version.txt

Reply:
Or just leave it empty, like we do it in Camel. I think that way it picks the latest published version. I can publish the jars for 0.0.4-SNAPSHOT as soon as I have some time, so it would work until we have a release.

Reply:
@orpiske I guess we want to declare default value to only final versions (not snapshots), but right now, `project.version` is snapshot. I thought we would like to update it only to new final versions, together with release procedure.

Reply:
Yeah, makes sense. It would be good to find a way to do that automatically, as the release process itself is already a bit complicated. I think it's OK for now, but would be good to find a way to automate this.